docs/section23/Rule23-13.md

```diff
@@ -0,0 +1,44 @@
+
+# Airside System - Rule 23-13 
+
+**Schema Version:** 0.0.34  
+**Mandatory Rule:** True  
+**Rule ID:** 23-13  
+**Rule Description:** If proposed design includes humidistatic controls then the baseline shall use mechanical cooling for dehumidification and shall reheat to avoid overcooling.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 23 Air-side  
+**90.1 Section Reference:** G3.1.3.18 Dehumidification (Systems 3 through 8 and 11, 12, and 13)  
+**Data Lookup:** None  
+**Evaluation Context:** HVAC system  
+
+**Applicability Checks:**  
+
+1. P-RMR is modeled with at least one air-side system that includes humidistatic controls.  
+
+**Function Calls:**  
+
+1. get_dict_of_zones_and_terminal_units_served_by_hvac_sys
+
+
+**Applicability Checks:**  
+- get a list of the hvac systems and the zones they served in the P_RMR: `hvac_systems_and_zones_p = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(P_RMR)`
+- look through each hvac system in the proposed: `for hvac_p_id in hvac_systems_and_zones_p:`
+    - get the proposed hvac sytem: `hvac_p = get_object_by_id(hvac_p_id)`
+    - set a boolean has_humidifcation_controls equal to false: `has_humidifcation_controls = FALSE`
+    - ZONE_LIST
+    - get the heating system: `heating_system_p = hvac_p.heating_system`
+    - check to see if the heating system has a humidification type other than NONE, if so, there are humidification controls, set has_humidifcation_controls to TRUE: `if heating_system_p.humidification_type != NONE: has_humidifcation_controls = TRUE`
+    - get the cooling system: `cooling_system_p = hvac_p.cooling_system`
+    - check to see if the cooling system has a humidification type other than NONE, if so, there are humidification controls set has_humidifcation_controls = TRUE: `if cooling_system_p.humidification_type != NONE: has_humidifcation_controls = TRUE`
+    - if there are humidity controls, we will now check for zone humidity setpoint schedules: `if has_humidifcation_controls:`
+        - look through each zone served by the system: `for zone_id in hvac_systems_and_zones_p[hvac_p_id]["ZONE_LIST"]:`
+            - get the zone: `zone_p = get_object_by_id(zone_id)`
+            - if the zone_p.minimum_humidity_setpoint_schedule exists, the rule is applicable, return UNDETERMIEND: `if zone_p.minimum_humidity_setpoint_schedule: UNDETERMINED`
```

Comment:
Recommend checking to see if any of the zones have humidity setpoints for the hvac system and then returning to the hvac system level to issue your outcome. I recommend including a message with the UNDETERMINED outcome regarding what the reviewer needs to check with the HVAC system ID.

Reply:
changed.  Rule is now applicable at the HVAC level and gives the modeler a list of zones that trigger the applicable result